tests/functional/lib/helpers.js

```diff
@@ -20,15 +20,6 @@ define(
       return params ? base + params : base;
     };
 
-    function takeScreenshot() {
```

Comment:
(In the future, since this change is unrelated to getting forks to pass tests, it's OK to include a separate commit for it.)